test/e2e/catalog_e2e_test.go

```diff
@@ -247,23 +251,23 @@ var _ = Describe("Catalog", func() {
 
 		// Attempt to get the catalog source before creating install plan
 		fetchedInitialCatalog, err := fetchCatalogSourceOnStatus(crc, mainCatalogName, testNamespace, catalogSourceRegistryPodSynced)
-		require.NoError(GinkgoT(), err)
+		Expect(err).ShouldNot(HaveOccurred())
 
 		// Get initial configmap
 		configMap, err := c.KubernetesInterface().CoreV1().ConfigMaps(testNamespace).Get(context.TODO(), fetchedInitialCatalog.Spec.ConfigMap, metav1.GetOptions{})
-		require.NoError(GinkgoT(), err)
+		Expect(err).ShouldNot(HaveOccurred())
 
 		// Check pod created
 		initialPods, err := c.KubernetesInterface().CoreV1().Pods(testNamespace).List(context.TODO(), metav1.ListOptions{LabelSelector: "olm.configMapResourceVersion=" + configMap.ResourceVersion})
-		require.NoError(GinkgoT(), err)
-		require.Equal(GinkgoT(), 1, len(initialPods.Items))
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(len(initialPods.Items)).To(Equal(1))
```

Comment:
Missed one of these:
```suggestion
		Expect(initialPods.Items).To(HaveLen(1))
```